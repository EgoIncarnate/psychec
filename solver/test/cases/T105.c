// Just like 104.c, but the function name is `new', which is a keyword in C++
// but not in C.

void new(set* s, unsigned capacity) {
  s->vector = (unsigned*)malloc((1 + capacity / INT_BITS) * sizeof(unsigned));
  s->size = capacity;
}
