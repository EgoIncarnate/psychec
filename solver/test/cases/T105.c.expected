typedef  struct __set__ {unsigned int* vector;int size;} set ;
unsigned int INT_BITS ;
